css/properties/-webkit-column-progression.json

```diff
@@ -20,7 +21,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "7"
```

Comment:
✅ First seen in 6.2.8 (https://github.com/caugner/css-properties/commit/cac6f0b28b05b205b52cbe8943674b00958eb8d7), probably a backport.